e2e/worker/worker/index.js

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+const { SQSClient, ReceiveMessageCommand, DeleteMessageCommand } = require("@aws-sdk/client-sqs");
+const axios = require("axios");
+const client = new SQSClient({ region: process.env.AWS_DEFAULT_REGION });
+
+console.log(`COPILOT_QUEUE_URI: ${process.env.COPILOT_QUEUE_URI}`);
+
+const eventsQueue = process.env.COPILOT_QUEUE_URI;
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+(async () => {
+  console.log(`The queue url created is: ${eventsQueue}`);
+  while(true) {
+    try {
+      await sleep(300);
+      const out = await client.send(new ReceiveMessageCommand({
+        QueueUrl: eventsQueue,
+        WaitTimeSeconds: 10,
+      }));
+      console.log(`results: ${JSON.stringify(out)}`);
+
+      if (out.Messages === undefined || out.Messages.length === 0) {
+        continue;
+      }
+
+      const resp = await axios.post(`http://frontend.${process.env.COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080/ack`);
```

Comment:
Should we delete the message only after the post action is successful?

Reply:
I don't think it particularly matters 🤷 because if we fail, and the messages never get deleted then `app delete` will delete the queue anyway with all of its messages.
The order shouldn't matter that much as long as we only make this request after receiving a real message